packages/nodejs/src/client.ts

```diff
@@ -159,6 +170,52 @@ export class BaseClient implements Client {
     return demo(this.tracer())
   }
 
+  /**
+   * Sets up the AppSignal logger with the output based on the `log` config option. If
+   * the log file is not accessible, stdout will be the output.
+   */
+  private setUpLogger(): winston.Logger {
+    const logFilePath = this.config.logFilePath
+    const logLevel = String(this.config.data["logLevel"])
+    let transport
+
+    const logFormat = printf(({ level, message, timestamp }) => {
+      return `[${timestamp} (process) #${process.pid}][appsignal][${level}] ${message}`
+    })
+
+    if ((this.config.data["log"] = "file" && logFilePath)) {
+      transport = new winston.transports.File({ filename: logFilePath })
+    } else {
+      transport = new winston.transports.Console()
+    }
+
+    return winston.createLogger({
```

Comment:
It may be better to create a wrapper class for this, so we control our own interface.

Reply:
Yep it's on to do

Reply:
Ah ok, I misunderstood that then.